suzieq/engines/pandas/evpnVni.py

```diff
@@ -107,6 +107,10 @@ def get(self, **kwargs) -> pd.DataFrame:
                         .rename(columns={'macaddr': 'numMacs'}) \
                         .fillna({'numMacs': 0})
 
+        for x in ['vlan', 'vni']:
+            if x in df.columns:
+                df[x] = df[x].astype(int)
```

Comment:
Is it possible that the column `vni` has `NaN` values? If yes it could crash here.

Reply:
OK, will address that

Reply:
Made use of np.where to address this